.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
```

Comment:
Can this be included in the pyproject.toml?

Reply:
Unfortunately, it can't -- and there is a rather heated (and in the meantime also locked) discussion about it: https://gitlab.com/pycqa/flake8/-/issues/428#note_296413313

Reply:
if anyone wants to set up flakehell it could work.